regen-network/regen-web
# App 239 add not editable once published warning in create post

## Description

https://regennetwork.atlassian.net/browse/APP-239

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...


### How to test

1. In the `Project Page > Data Stream` or in the `Projects Page`, click on the **Create Post** button. Then, scroll to the bottom of the New Post form where the warning should be visible.

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
